dropwizard-logging/src/test/java/io/dropwizard/logging/FileAppenderFactoryTest.java

```diff
@@ -190,7 +190,10 @@ public void hasMaxFileSizeFixedWindow() throws Exception {
         RollingFileAppender<ILoggingEvent> appender = (RollingFileAppender<ILoggingEvent>) fileAppenderFactory.buildAppender(new LoggerContext());
 
         assertThat(appender.getRollingPolicy()).isInstanceOf(FixedWindowRollingPolicy.class);
+        assertThat(appender.getRollingPolicy().isStarted());
+        
         assertThat(appender.getTriggeringPolicy()).isInstanceOf(SizeBasedTriggeringPolicy.class);
+        assertThat(appender.getTriggeringPolicy().isStarted());
```

Comment:
Adding this line without the above change to `FileAppenderFactory` doesn't result in a test failure for me. How did you confirm that the triggering policy isn't started?